cmd/exec/exec.go

```diff
@@ -42,15 +43,24 @@ func ExecutePlan(commands ...string) {
 }
 
 func executeAndStopIfFailed(command *models.CommandContext, executeStart time.Time) {
-	command.PrintPhaseBanner()
-	err := command.CommandSession.Run()
-	if err != nil {
+	stdout, err := command.CommandSession.CombinedOutput()
+	if !viper.GetBool("quiet") {
+		command.PrintPhaseBanner()
```

Comment:
Let's print phase banner regardless of quiet flag - As you would be interested in seeing which phase failed amount all